### PR TITLE
More 1.7.0 doc updates

### LINKED ISSUE #458: [Docs] Kyverno Cosign Public Key not found
**Is your feature request related to a problem? Please describe current state:**
In the docs, in the [Verifying Kyverno image signatures using Cosign](https://kyverno.io/docs/installation/#verifying-kyverno-image-signatures-using-cosign) there is a link that points to organization public key for verification using cosign but unfortunately is [not found / 404](https://github.com/kyverno/kyverno/blob/main/cosign.pub)

**Describe the solution you'd like:**
Fix the link in the docs to the proper path of the cosign.pub

**Describe alternatives you've considered:**
Store the key outside of the kyverno repo, like in an static site for store utils like this public key.

---
**Contributor Resources**
- [Contributing Guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md)

### LINKED ISSUE #549: [1.7.0] Add imageRegistry info on Docker Hub example
### Description

From https://github.com/kyverno/website/pull/548#issuecomment-1146047729

Add the info that was shared in the linked Slack discussion.

### Slack discussion

https://kubernetes.slack.com/archives/CLGR9BJU9/p1653922382390759

### PR DESCRIPTION
**Pull request by chipzoller**
## Related issue #

Closes #458 
Closes #549 

Cherry pick needed to 1.7.0
## Proposed Changes

* Adds current cosign validation info
* Adds notes on `foreach` accepting array/string
* Adds note on imageData variable with Docker Hub official images
* Fixes Kyverno image verification procedure

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

File: content/en/docs/security/_index.md [hunk 1]
```diff
@@ -59,18 +59,46 @@ export COSIGN_REPOSITORY=ghcr.io/kyverno/signatures
...
+        "SignedEntryTimestamp": "MEQCIDnRtzZE3z9CEsE5uFvwSFgfwZqdoly3HjHdFYzdOY/4AiB2JtOUQxKbrC0BoMnEYR9HSGnK9yX+nW4P5KZ+Aw/jYw==",
+        "Payload": {
+          "body": "<snip>",
+          "integratedTime": 1654284265,
+          "logIndex": 2551796,
+          "logID": "c0d23d6ad406973f9559f3ba2d1ca01f84147d8ffc5b8445c224f98b9591801d"
+        }
+      },
+      "Issuer": "https://token.actions.githubusercontent.com",
+      "Subject": "https://github.com/kyverno/kyverno/.github/workflows/reuse.yaml@refs/heads/main",
```
Comment (samj1912) [inline content/en/docs/security/_index.md:95]: We should note that these two fields are the important ones. If the issuer and subject are not this, then it is an invalid image.

Comment (chipzoller) [reply content/en/docs/security/_index.md:95]: Captured in https://github.com/kyverno/website/pull/557/commits/c87597e407759e73161439a416bc6df6d0a84402

File: content/en/docs/security/_index.md [hunk 2]
```diff
@@ -59,20 +59,50 @@ export COSIGN_REPOSITORY=ghcr.io/kyverno/signatures
...
+      "ref": "0b7b2458eb46d157e27fcc51bf3bc67d99d5cab9",
+      "repo": "kyverno/kyverno",
+      "workflow": "image"
+    }
+  }
+]
 ```
 
+Note that the important fields to verify in the output are `optional.Issuer` and `optional.Subject`.
```
Comment (samj1912) [inline content/en/docs/security/_index.md:104]: Could we also add something along the lines of - If the subject and issuer do not match the above then it's an image that didn't originate from our repository?

Comment (chipzoller) [reply content/en/docs/security/_index.md]: Added disclaimer in https://github.com/kyverno/website/pull/557/commits/313374fbb4846730ee48cb35ae15b5c1c323c92d

**Pull request merged**